students/preactive/Lesson2/fizzbuzz.py

```diff
@@ -0,0 +1,10 @@
+for i in range(1,100):
```

Comment:
This should be for i in range(1, 101) for 1 through 100 inclusive.

Reply:
Fixed